pkg/rsyslog/init.sh

```diff
@@ -1,3 +1,15 @@
 #!/bin/sh
 
+RSYSLOG_LOG_DIR=/persist/syslog/
+RSYSLOG_WORK_DIR=/persist/rsyslog
+if [ ! -d "$RSYSLOG_WORK_DIR" ]; then
+  mkdir -p $RSYSLOG_WORK_DIR
+  chmod 644 $RSYSLOG_WORK_DIR
+fi
+if [ ! -d "$RSYSLOG_LOG_DIR" ]; then
+  mkdir -p $RSYSLOG_LOG_DIR
+  chmod 644 $RSYSLOG_LOG_DIR
+fi
 IMGP=IMGA /usr/sbin/rsyslogd
+sleep 5
+/usr/bin/logread -F | logger &
```

Comment:
So here's a much bigger question we need to figure out -- whether rsyslog container pulls from sources or other containers push into it. With the push model, this last statement will move to a different container. This will also allow us to solve the container dependency issue.

Honestly, this part of the design may be worth to discuss on eve-tsc@

Reply:
@rvs I did not understand the push/pull models completely.
1) By pull, do you mean rsyslogd container running logread command to pull logs from memlogd?
2) By push, do you mean logread command running in a separate container pull logs and then pushing them to rsyslog running in a different container?

All the other log sources (eve services, dnsmasq etc) will work in push model. Can they work differently?